lib/season_stat_team.rb

```diff
@@ -28,6 +28,16 @@ def get_team_name(team_id)
     team_name_by_id.team_name
   end
 
+  def get_tackles_by_team_season(team_id, season)#refactor to a reduce
+    team_tackles = 0
+    get_season_game_teams(season).each do |game_team|
+      if game_team.team_id.to_s == team_id
+        team_tackles += game_team.tackles
+      end
```

Comment:
another helper method that is similar to all others, this grabs the tackles and throws it in an accumulator

Reply:
I already like the way this looks, but make sure we get rid of comments before we push the final product.